src/Inkluzitron/Modules/BdsmTestOrg/BdsmModule.cs

```diff
@@ -22,35 +21,44 @@
 using System.Reflection;
 using Inkluzitron.Services;
 using Inkluzitron.Utilities;
+using Inkluzitron.Models;
 
 namespace Inkluzitron.Modules.BdsmTestOrg
 {
     [Name("BDSMTest.org")]
     [Group("bdsm")]
     [Summary("Dotaz na kategorie může mít následující podoby:\n`dom sub` == `dom>50 sub>50` == `+dom +sub`\n`dom -switch` == `dom switch<50`")]
-    public class BdsmModule : ModuleBase
+    public sealed class BdsmModule : ModuleBase, IDisposable
     {
         static private readonly Regex TestResultLinkRegex = new(@"^https?://bdsmtest\.org/r/([\d\w]+)");
 
         private BotDatabaseContext DbContext { get; set; }
         private DatabaseFactory DatabaseFactory { get; }
         private ReactionSettings ReactionSettings { get; }
         private BdsmTestOrgSettings Settings { get; }
-        private GraphPaintingService GraphPainter { get; }
         private IHttpClientFactory HttpClientFactory { get; }
-        public UserBdsmTraitsService BdsmTraitsService { get; }
+        private UserBdsmTraitsService BdsmTraitsService { get; }
+        private GraphPaintingService GraphPaintingService { get; }
+        private BdsmGraphPaintingStrategy GraphPaintingStrategy { get; }
 
         public BdsmModule(DatabaseFactory databaseFactory,
             ReactionSettings reactionSettings, BdsmTestOrgSettings bdsmTestOrgSettings,
             GraphPaintingService graphPainter, IHttpClientFactory factory,
-            UserBdsmTraitsService bdsmTraitsService)
+            UserBdsmTraitsService bdsmTraitsService, FontService fontService)
         {
             DatabaseFactory = databaseFactory;
             ReactionSettings = reactionSettings;
             Settings = bdsmTestOrgSettings;
-            GraphPainter = graphPainter;
             HttpClientFactory = factory;
             BdsmTraitsService = bdsmTraitsService;
+            GraphPaintingService = graphPainter;
+            GraphPaintingStrategy = new BdsmGraphPaintingStrategy(fontService);
```

Comment:
Nebylo by lepší to nechat na starosti kontejneru?

Reply:
Necháno na starosti kontejneru.